src/Consumer.cc

```diff
@@ -63,7 +63,9 @@ void MessageListenerProxy(Napi::Env env, Napi::Function jsCallback, MessageListe
   Consumer *consumer = data->consumer;
   delete data;
 
-  jsCallback.Call({msg, consumer->Value()});
+  if (consumer) {
```

Comment:
Do we need to print some error messages when the consumer is a null value?

Reply:
I'm not sure if users lose to handle some messages when the consumer is null.

Reply:
I think the controversial point is whether should we skip the null consumer.
- Yes: might lose messages (not sure)
- No: fast fail

Adding logs doesn't solve anything. There is no logger in NodeJS client, printing logs to console is meaningless and helpless as I said.

Reply:
We need more info to reproduce the segmentation fault. But for now, a safe solution is to avoid accessing a null consumer.